src/timelines/breakTrial.js

```diff
@@ -3,17 +3,17 @@ import { lang } from '../config/main'
 import breakScreen from '../trials/breakScreen'
 import buildCountdown from '../trials/countdown'
 
-const breakTrial = () => {
+const breakTrial = (iBreak) => {
 
-  let timeline = [
-    breakScreen(),
-    buildCountdown(lang.countdown.post_break_resume, 3)
-  ]
+    let timeline = [
+        breakScreen(iBreak),
+        buildCountdown(lang.countdown.post_break_resume, 3)
```

Comment:
looks good! am a lil confused bc i assumed you added iBreak already but if this is a PR combining both the break stuff & conditional timeline then that's fine

Reply:
interesting, might be because I made a new branch